VideoLocker/build.gradle

```diff
@@ -234,6 +234,8 @@ android {
             if (fabric != null) {
                 ext.enableCrashlytics = fabric.get('FABRIC_KEY') != null
             }
+            def platformName = config.get('PLATFORM_NAME')
+            resValue "string", "app_name", platformName
```

Comment:
It was probably none because it wasn't in your config. I'm updating all the configs to include one.